examples/javascript/test/browser/chromeSpecificCaps.spec.js

```diff
@@ -47,6 +47,17 @@ suite(function (env) {
       await driver.quit();
     });
 
+    it('Add Extension', async function () {
+      const options = new Chrome.Options();
+      let driver = await env
+        .builder()
+        .setChromeOptions(options.addExtensions(['/path/to/extension.crx']))
```

Comment:
The tests runs in CI, so i believe these will fail! you may need to exclude test in ci/run the test with a valid extension!